{{cookiecutter.project_dirname}}/scripts/ci_pact.sh

```diff
@@ -3,13 +3,17 @@
 if [ "${VAULT_ADDR}" != "" ]; then
   apk update && apk add curl jq
 
-  export VAULT_TOKEN=`curl --silent --request POST --data "role=pact" --data "jwt=${CI_JOB_JWT_V2}" ${VAULT_ADDR%/}/v1/auth/gitlab-jwt/login | jq -r .auth.client_token`
+  vault_token=$(curl --silent --request POST --data "role=pact" --data "jwt=${CI_JOB_JWT_V2}" "${VAULT_ADDR%/}"/v1/auth/gitlab-jwt/login | jq -r .auth.client_token)
 
-  pact_secrets=`curl --silent --header "X-Vault-Token: ${VAULT_TOKEN}" ${VAULT_ADDR%/}/v1/${PROJECT_SLUG}/pact | jq -r .data`
+  pact_secrets=$(curl --silent --header "X-Vault-Token: ${vault_token}" "${VAULT_ADDR%/}"/v1/"${PROJECT_SLUG}"/pact | jq -r .data)
 
-  export PACT_BROKER_BASE_URL=`echo ${pact_secrets} | jq -r .pact_broker_base_url`
-  export PACT_BROKER_PASSWORD=`echo ${pact_secrets} | jq -r .pact_broker_password`
-  export PACT_BROKER_USERNAME=`echo ${pact_secrets} | jq -r .pact_broker_username`
+  PACT_BROKER_BASE_URL=$(echo "${pact_secrets}" | jq -r .pact_broker_base_url)
+  PACT_BROKER_PASSWORD=$(echo "${pact_secrets}" | jq -r .pact_broker_password)
+  PACT_BROKER_USERNAME=$(echo "${pact_secrets}" | jq -r .pact_broker_username)
+
+  export PACT_BROKER_BASE_URL
+  export PACT_BROKER_PASSWORD
```

Comment:
consider to add an `unset` for sensitive environment variables at point where they become no longer necessary

Reply:
GitLab runner deals with this already, there are a lot of sensitive env vars involved in a single run of a CI job (e.g. the JWT token itself), and they are all handled correctly by the runner. I would not worry about this or anything of the sort.